win32ss/user/ntuser/focus.c

```diff
@@ -1287,15 +1287,15 @@ UserSetActiveWindow( _In_opt_ PWND Wnd )
       !(gpqForegroundPrev->spwndActivePrev->state2 & WNDS2_BOTTOMMOST) &&
        (Wnd = VerifyWnd(gpqForegroundPrev->spwndActivePrev)) != NULL )
   {
-     TRACE("USAW:PAW hwnd %p\n",Wnd?Wnd->head.h:NULL);
+     TRACE("USAW:PAW hwnd %p\n", Wnd ? UserHMGetHandle(Wnd) : NULL);
```

Comment:
Here technically, Wnd cannot be NULL (see the verifywnd condition above)
```suggestion
     TRACE("USAW:PAW hwnd %p\n", UserHMGetHandle(Wnd));
```